subsys/usb/class/bluetooth.c

```diff
@@ -188,7 +188,7 @@ static void acl_read_cb(u8_t ep, int size, void *priv)
 		net_buf_unref(buf);
 	}
 
-	buf = net_buf_alloc(&acl_tx_pool, K_NO_WAIT);
+	buf = net_buf_alloc(&acl_tx_pool, K_FOREVER);
```

Comment:
instead of logging error, use __ASSERT_NO_MSG, that is how it is done in bluetooth subsystem.

Reply:
Will fix this.

Reply:
Yep. Assert should be used for things which indicate a bug somewhere, and getting NULL when you have K_FOREVER is definitely a bug.